sakai-evaluation-tool/src/webapp/content/templates/report_view.html

```diff
@@ -42,13 +43,22 @@
          	<li style="float:left"><a rsf:id="summary-link" href="summary.html">Summary</a></li>
          	<li style="float:left"><a rsf:id="report-groups-title" href="report_groups.html">Choose Report Groups</a></li>
          	<li rsf:id="view-report-title" style="float:left" class="lastCrumb">View Report</li>
-            <li style="float:right" class="lastCrumb"><a rsf:id="pdfResultsReport" href="#" target="_blank" class="exportAction">Export pdf</a></li>
-            <li style="float:right" class="lastCrumb"><a rsf:id="xlsResultsReport" href="#" target="_blank" class="exportAction">Export xls</a></li>
-            <li style="float:right" class="lastCrumb"><a rsf:id="csvResultsReport" href="#" target="_blank" class="exportAction">Export csv</a></li>
             <li style="float:right" class="lastCrumb"><a rsf:id="viewItemsFilterLink" href="#" class="goToList">View Written Responses Only</a></li>
-            <li style="float:right" class="lastCrumb"><a rsf:id="exportListOfEvaluationTakers" href="#" target="_blank" class="exportAction">Export csv of evaluation takers</a></li>
-         </ul>
-		 
+        </ul>
+
+        <form rsf:id="evalReportForm" name="eval_report_form" method="post">
+           <input rsf:id="xlsResultsReport" name="xlsResultsReport" type="submit" value="Export xls" />
```

Comment:
I'm thinking instead of changing the text on the CSV button, we could provide an explanation blurb beside the new report checkbox (toggle) to explain what the differences are, and mention that in the context of a CSV export with the new report style, you will be presented with a .zip file rather than a single .csv file.